Source/WebCore/css/CSSSelector.h

```diff
@@ -26,6 +26,7 @@
 #include "RenderStyleConstants.h"
 #include <wtf/EnumTraits.h>
 #include <wtf/FixedVector.h>
+#include <wtf/text/TextStream.h>
```

Comment:
Does `operator<<` need to be inlined? If not, we can avoid including `TextStream.h` here, and use a forward declaration instead; this is good to resist.

Reply:
It really doesn't, will move to .cpp